packages/extension-api/src/extension-api.d.ts

```diff
@@ -2425,4 +2425,31 @@ declare module '@podman-desktop/api' {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     export function setValue(key: string, value: any, scope?: 'onboarding'): void;
   }
+
+  export interface CliToolOptions {
+    id: string;
+    name: string;
+    displayName: string;
+    markdownDescription: string;
+    images: ProviderImages;
+  }
+
+  export type CliToolState = 'registered';
+
+  export interface CliTool extends Disposable {
```

Comment:
Are you able to add comments similar to other parts of extension-api.d.ts? This will help others when implementing their own extension on how to use the CLI functionality and this functionality is regarding creating update / download / management sections for podman desktop.

Reply:
done.